spec/factories/events.rb

```diff
@@ -0,0 +1,55 @@
+FactoryBot.define do
+  factory :event do
+    association(:move)
+    event_type { 'move_created' }
+    client_timestamp { Time.now.utc + rand(-60..60).seconds } # NB: the client_timestamp will never be perfectly in sync with system clock
```

Comment:
Is this an ecological validity of our fixtures thing? Doesn't seem necessary/tested/important for any of our examples?

Reply:
It will become important as the client_timestamp is a critical field out with our control for assembling the order of events. We will rely on clients to accurately keep their clocks in sync. This code models the fact the `client_timestamp` will be a few seconds either side of the `created_at` timestamp, which will become important further down the line.